static.config.js

```diff
@@ -653,6 +657,34 @@ const makeAllPages = async langCode => {
   const siteStructure = await client.request(siteStructureQuery);
   let parsedStructure = JSON.parse(siteStructure.siteStructure.structureJson);
 
+  // This is really something that should happen in joplin,
+  // but let's just use janis to do it for now
+  if (incrementalPageId) {
+    // First let's find all of the parent/grandparent ids we need
+    // to rebuild titles on links etc.
+    let idsToRebuild = [incrementalPageId];
+
+    for (const pageAtUrlInfo of parsedStructure) {
+      if (pageAtUrlInfo.id === incrementalPageId) {
+        if (pageAtUrlInfo.parent_department) {
+          idsToRebuild.push(pageAtUrlInfo.parent_department);
+        }
+
+        if (pageAtUrlInfo.parent_topic) {
+          idsToRebuild.push(pageAtUrlInfo.parent_topic);
+        }
+
+        if (pageAtUrlInfo.grandparent_topic_collection) {
+          idsToRebuild.push(pageAtUrlInfo.grandparent_topic_collection);
+        }
+      }
+    }
+
+    parsedStructure = parsedStructure.filter(paui =>
```

Comment:
The full name would be easier to understand than an acronym

Reply:
renamed it